pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: sass
-version: 1.64.0-dev
```

Comment:
Since 1.64.0 hasn't been released (you can tell because of the `-dev`), there's no need to bump it any further.